pages/add/index.js

```diff
@@ -1,17 +1,18 @@
 import AddEntryForm from "@/components/AddEntryForm";
 import EntriesContainer from "@/components/EntriesContainer";
 import Heading from "@/components/PageHeading";
+import { MainContent } from "@/components/StyledComponents/MainContentSection";
```

Comment:
```suggestion
import { MainContent } from "@/components/StyledComponents/MainContentSection";
```
Just keep this line in mind when you do choose to rename the file.